Google.Ads.GoogleAds/examples/AdvancedOperations/AddAdCustomizer.cs

```diff
@@ -126,347 +115,212 @@ public void Run(GoogleAdsClient client, long customerId, long[] adGroupIds)
-            };
-
-            MutateFeedsResponse response =
-                feedService.MutateFeeds(customerId.ToString(), new[] { feedOperation });
-
-            string feedResourceName = response.Results[0].ResourceName;
-            Console.WriteLine($"Added feed with resource name '{feedResourceName}'.");
-            return feedResourceName;
+            return customizerAttributeResourceName;
         }
         // [END add_ad_customizer]
 
         /// <summary>
-        ///  Retrieves all the attributes for a feed and returns them in a map using the
-        ///  attribute names as keys.
+        /// Creates a price customizer attribute ad returns its resource name.
         /// </summary>
         /// <param name="client">The Google Ads client.</param>
-        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
-        /// <param name="feedResourceName">The resource name of the feed.</param>
-        /// <returns>The attributes of the feed.</returns>
+        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>        
+        /// <param name="customizerName">The name of the customizer to create.</param>
         // [START add_ad_customizer_1]
-        private Dictionary<string, FeedAttribute> GetFeedAttributes(GoogleAdsClient client,
-                    long customerId, string feedResourceName)
+        private string CreatePriceCustomizerAttribute(GoogleAdsClient client, long customerId,
+            string customizerName)
         {
-            // Get the GoogleAdsServiceClient.
-            GoogleAdsServiceClient googleAdsService =
-                client.GetService(Services.V11.GoogleAdsService);
-
-            string query = $"SELECT feed.attributes, feed.name FROM feed WHERE " +
-                $"feed.resource_name = '{feedResourceName}'";
-
-            SearchGoogleAdsRequest request = new SearchGoogleAdsRequest()
+            // Get the customizer attribute service.
+            CustomizerAttributeServiceClient customizerAttributeService = 
+                client.GetService(Services.V11.CustomizerAttributeService);
+    
+            // Creates a price customizer attribute. The customizer attribute name is
+            // arbitrary and will be used as a placeholder in the ad text fields.
+            CustomizerAttribute priceAttribute = new CustomizerAttribute()
             {
-                CustomerId = customerId.ToString(),
-                Query = query
+                Name = customizerName,
+                Type = CustomizerAttributeType.Price
+            };
+    
+            CustomizerAttributeOperation priceAttributeOperation = new CustomizerAttributeOperation(){
+                Create = priceAttribute
             };
 
-            Dictionary<string, FeedAttribute> feedAttributes =
-                new Dictionary<string, FeedAttribute>();
+            List<string> customizerAttributeResourceNames = new List<string>();
```

Comment:
@Raibaz this isn't used and should be removed.

Reply:
Thanks for spotting this, addressed in #443.